app/assets/images/payment_icons/line_pay.svg

```diff
@@ -0,0 +1 @@
.509-1.057 2.389-.283.88-.758 1.955-1.057 2.389-.299.434-.544 1.154-.544 1.6-.001.446-.351 1.383-.777 2.082-.426.699-.894 1.907-1.04 2.686-.146.778-.512 1.619-.813 1.869-.301.25-.67 1.108-.82 1.906-.15.798-.604 1.995-1.009 2.66-.405.664-.738 1.573-.739 2.019 0 .446-.251 1.168-.556 1.603-.305.436-.79 1.601-1.077 2.589-.288.989-.74 2.157-1.006 2.597-.266.44-.603 1.373-.748 2.072-.146.7-.505 1.472-.798 1.715-.293.243-.656 1.11-.808 1.927-.152.817-.508 1.756-.791 2.086-.284.33-.729 1.266-.991 2.079-.621 1.936-1.034 1.99-14.183 1.84-12.01-.138-11.527.033-10.654-3.78.295-1.286.868-2.9 1.273-3.586.406-.686.739-1.586.741-2 .001-.414.36-1.293.797-1.953.437-.66.798-1.47.801-1.8.003-.33.449-1.5.99-2.6.541-1.1.99-2.383.996-2.851.007-.468.271-1.066.587-1.328.316-.262.7-1.049.854-1.749.153-.699.615-1.899 1.025-2.666.411-.767.747-1.719.747-2.116s.386-1.244.857-1.882c1.258-1.702 1.273-6.294.026-7.981-.486-.657-.884-1.517-.885-1.911-.002-.394-.283-1.253-.625-1.909-.831-1.593-2.463-5.742-2.789-7.091-.144-.596-.51-1.289-.813-1.54-.302-.251-.671-1.113-.818-1.916-.148-.803-.603-1.889-1.011-2.414-.408-.525-.742-1.355-.742-1.846 0-.491-.36-1.35-.8-1.909-.44-.56-.806-1.349-.813-1.754-.006-.405-.455-1.637-.996-2.737-.541-1.1-.985-2.242-.987-2.538-.002-.296-.245-1.016-.539-1.6-1.426-2.829-2.256-4.769-2.509-5.862-.153-.66-.611-1.629-1.017-2.154-.406-.525-.74-1.335-.742-1.8-.001-.465-.36-1.386-.797-2.046-.437-.66-.797-1.56-.799-2-.002-.44-.342-1.355-.756-2.033-.414-.679-.875-1.791-1.024-2.473-.15-.681-.531-1.453-.847-1.715-.316-.262-.574-.878-.574-1.368 0-.491-.45-1.738-1-2.773-.55-1.034-1-2.136-1-2.449 0-.314-.359-1.159-.799-1.879-.439-.721-.799-1.56-.8-1.866-.001-.565-1.057-3.043-2.438-5.723-.42-.814-.763-1.721-.763-2.016 0-.295-.35-1.221-.777-2.059-.427-.838-.889-2.034-1.026-2.657-.136-.624-.498-1.341-.803-1.594-.776-.644-1.175-3.709-.618-4.749.444-.83.661-.846 11.574-.846h11.121l1.431 1.1m-96.304 21.934c-4.827 2.464-9.398 7.578-9.398 10.514 0 .371-.305 1.109-.677 1.641-1.107 1.58-1.261 9.333-.232 11.659.433.978.913 2.448 1.066 3.266.154.817.433 1.486.622 1.486s.524.52.745 1.155c.521 1.493 4.628 5.6 6.121 6.121.635.221 1.155.558 1.155.748 0 .19.765.54 1.7.777.935.238 2.132.65 2.659.916 2.429 1.224 11.739.351 14.441-1.354.33-.209 1.14-.635 1.8-.947 3.085-1.461 8.2-8.142 8.2-10.711 0-.43.286-1.764.634-2.964.861-2.961.222-9.786-1.154-12.341a84.97 84.97 0 01-1.188-2.348c-.417-.852-1.804-2.562-3.084-3.8-3.247-3.143-3.834-3.593-5.098-3.91-.61-.152-1.65-.606-2.31-1.008-1.011-.616-2.083-.731-6.804-.733l-5.603-.001-3.595 1.834m15.603-.2c4.153 1.46 10.199 6.285 10.199 8.14 0 .241.371 1.129.824 1.974 2.496 4.659 1.594 16.522-1.54 20.24-2.825 3.351-4.677 4.842-7.884 6.345-4.466 2.094-13.383 1.882-16.662-.396-.584-.405-1.327-.737-1.651-.737-.781 0-5.424-4.675-6.158-6.2-.317-.66-.834-1.65-1.147-2.2-.314-.55-.573-1.382-.576-1.85-.003-.467-.276-1.615-.606-2.55-.646-1.83-.824-7.015-.253-7.367.191-.118.469-1.128.617-2.245 1.329-10.023 14.26-16.871 24.837-13.154" fill="#84dcac"/></g></svg>
```

Comment:
1- Is it possible to add a white background? Otherwise, if a merchant store is changing their background to black, the icon will likely have a black background as well

[![alt](https://screenshot.click/21-49-7zj9w-nh27n.png)](https://screenshot.click/21-49-7zj9w-nh27n.png)

2- Is it possible to add a border to follow other icons? Example: https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg

3- Can we remove all `id` attributes that are unused? `pi-line_pay-path0`, `pi-line_pay-path1`, `pi-line_pay-path2`, `pi-line_pay-path3` and `pi-line_pay-path4`

4- The viewBox needs to be `viewBox="0 0 38 24"`

Reply:
Hey, thanks for the feedback!
1. I added the code to add a white background. No idea if this worked.
2. I added the code to add a 70% opacity black border. No idea if this worked.
3. I removed all `path` ids.
4. I can't seem to edit `viewBox`s without destroying the SVGs. (before and after edit below)
<img width="100" alt="Screen Shot 2020-07-22 at 14 11 42" src="https://user-images.githubusercontent.com/1557529/88137294-351ae500-cc26-11ea-964b-1fa258129b83.png"><img width="100" alt="Screen Shot 2020-07-22 at 14 11 50" src="https://user-images.githubusercontent.com/1557529/88137299-39470280-cc26-11ea-91e1-c4a9b42ddcf5.png">

Reply:
I see the white background and the border is here only if we modify the `viewBox` to 38 and 24.
[video demonstration](https://screenshot.click/22-06-91zay-t27zk.mp4)

Do you think we can ask the designers to resize to a smaller size?

Reply:
> Do you think we can ask the designers to resize to a smaller size?

Hmm, which designers would this be? Legit just sourcing SVGs from websites at the moment. 😅

Reply:
Sorry, I thought the designers provided you the SVGs.
If not, would it be possible to modify the SVG directly with the correct format using Sketch or something similar?

Reply:
All good. @yishus was able to jump in and help out with some SVG editing. 🙌